datadog_checks_downloader/datadog_checks/downloader/cli.py

```diff
@@ -116,10 +122,26 @@ def download():
                 sys.exit(1)
 
     tuf_downloader = TUFDownloader(
-        repository_url_prefix=repository_url_prefix, root_layout_type=root_layout_type, verbose=verbose
+        repository_url_prefix=repository_url_prefix,
+        root_layout_type=root_layout_type,
+        verbose=verbose,
+        disable_verification=args.unsafe_disable_verification,
     )
+
+    return tuf_downloader, standard_distribution_name, version, ignore_python_version
+
+
+def run_downloader(tuf_downloader, standard_distribution_name, version, ignore_python_version):
     wheel_relpath = tuf_downloader.get_wheel_relpath(
         standard_distribution_name, version=version, ignore_python_version=ignore_python_version
     )
     wheel_abspath = tuf_downloader.download(wheel_relpath)
     print(wheel_abspath)  # pylint: disable=print-statement
+
+
+# Public module functions.
```

Comment:
BTW the convention in Python is to preface private functions with `_`. You can do this and get rid of this section comment.

Reply:
I agree, but note that this comment, and the function names, were not introduced by this PR: this PR merely extracts a part of existing functions.

If we want to refactor the existing code towards better function names, I'd rather do it in another PR, is this OK with you?

Reply:
imo the refactoring in this case is small enough to be rolled into this PR, but it can also be done as a separate one, up to you.